ver_03/script.js

```diff
@@ -0,0 +1,306 @@
+  playerTwoRoll.push(diceRoll_1, diceRoll_2);
+  console.log(playerTwoRoll);
+  return playerTwoChoice;
+};
+
+var playerTwoDiceDecisionK = function () {
+  var diceRoll_1 = playerTwoRoll[0];
+  var diceRoll_2 = playerTwoRoll[1];
+
+  var playerTwoDecision =
+    "Player Two, you have chosen to swap the dice." +
+    "<br><br>New dice one: " +
+    diceRoll_1 +
+    " <br>& " +
+    " <br>New dice two: " +
+    diceRoll_2 +
+    " <br><br>If Player One has completed their turn, <br>type <u><b>'result'</b></u> to get the comparison";
+
+  return playerTwoDecision;
+};
+
+var playerTwoDiceDecisionS = function () {
+  var diceTemp = playerTwoRoll[0];
+  playerTwoRoll[0] = playerTwoRoll[1];
+  playerTwoRoll[1] = diceTemp;
+
+  var diceRoll_1 = playerTwoRoll[0];
+  var diceRoll_2 = playerTwoRoll[1];
+
+  var playerTwoDecision =
+    "Player Two, you have chosen to swap the dice." +
+    "<br><br>New dice one: " +
+    diceRoll_1 +
+    "<br>& " +
+    "<br>New dice two: " +
+    diceRoll_2 +
+    " <br><br>If Player One has completed their turn, <br>type <u><b>'result'</b></u> to get the comparison.";
+
+  return playerTwoDecision;
+};
+
+var compareResults = function () {
+  var playerOneDice = JSON.stringify(playerOneRoll);
+  var playerTwoDice = JSON.stringify(playerTwoRoll);
+  playerOneDice = Number.playerOneDice;
+  playerTwoDice = Number.playerTwoDice;
+  console.log(playerOneDice, playerTwoDice);
+  if (playerOneDice > playerTwoDice) {
+    comparisonResult =
+      "Player One's roll of " +
+      playerOneDice +
+      " <br> is greater than <br>Player Two's roll of " +
+      playerTwoDice +
+      "<br><br>Player One Wins this round.";
+
+    playerOneWinStatusCounter += 1;
+    playerTwoLoseStatusCounter += 1;
+  } else if (playerOneDice < playerTwoDice) {
+    comparisonResult =
+      "Player One's roll of " +
+      playerOneDice +
+      " <br> is lesser than <br>Player Two's roll of " +
+      playerTwoDice +
+      "<br><br>Player Two Wins this round.";
+
+    playerTwoWinStatusCounter += 1;
+    playerOneLoseStatusCounter += 1;
+  } else {
+    comparisonResult =
+      "Player One's roll of " +
+      playerOneDice +
+      " <br> is the same as <br><br>Player Two's roll of " +
+      playerTwoDice +
+      "<br><br>both players tie this round.";
+
+    playerTieStatus += 1;
+  }
+
+  turnCounter += 1;
+  return comparisonResult;
+};
+
+// Function for random dice roller
+var randomDiceRoll = function () {
+  var randomDiceRollOutcome = Math.floor(Math.random() * 6) + 1;
+  // console.log("dice roll = " + 1);
+
+  return randomDiceRollOutcome;
+};
+
+var emptyChecker = function () {
+  var pIC1 = player1InputCheck;
+  var pIC2;
+  if (
+    (pIC1 || pIC2) == "" ||
+    (pIC1 || pIC2) != "keep" ||
+    (pIC1 || pIC2) != "swap" ||
+    pIC != "result"
+  ) {
+    return "Please enter the correct choice.";
+  }
```

Comment:
A little curious what `pIC2` does, since you are simply instantiating an empty variable here. The conditions are also almost definitely wrong, as you are comparing a `boolean` with string. Remember to do the conditions more explicitly for example:
```
pIC1 == "" || pIC2 == ""
```
And of course, variable namings can be better here as nobody would know what `pIC` stands for